Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -157,6 +158,13 @@ class BoolSetting: Setting {
         control.addTarget(self, action: "switchValueChanged:", forControlEvents: UIControlEvents.ValueChanged)
         control.on = prefs.boolForKey(prefKey) ?? defaultValue
         cell.accessoryView = control
+        cell.detailTextLabel?.snp_makeConstraints { make in
```

Comment:
This makes me nervous. I don't know about adding additional constraints to a existing views constructed by the OS like in this case with UITableView. It seems that these constraints were added to align the title/subtitle labels but since we don't set a constraint on the title label different locales would make these labels not line up.